.projen/tasks.json

```diff
@@ -9,9 +9,6 @@
         },
         {
           "exec": "yarn nx run-many --target=build --output-style=stream --nx-bail"
```

Comment:
Note I forgot to npx projen on main on the last PR merge. This is why this is a diff - it was not a manual change to this file